plugins/polio/js/src/domains/VaccineModule/SupplyChain/Details/VaccineRequestForm/VaccineRequestForm.tsx

```diff
@@ -25,6 +25,10 @@ import { useSharedStyles } from '../shared';
 
 type Props = { className?: string; vrfData: any };
 
+export const isFieldDisabledEdit = (inputData: any | undefined): boolean => {
+    return inputData ? !inputData.can_edit : false;
+};
+
```

Comment:
I would just declare the variable in the component:
`const isFieldDisabledEdit = inputData ? !inputData.can_edit : false;`

Since it's a boolean, you don't even have to put it in a `useMemo`